jaxrs/src/main/java/org/killbill/billing/jaxrs/json/AuditLogJson.java

```diff
@@ -36,24 +39,31 @@
     private final String comments;
     private final String userToken;
 
+    private final ObjectType objectType;
+    private final UUID objectId;
+
     @JsonCreator
     public AuditLogJson(@JsonProperty("changeType") final String changeType,
                         @JsonProperty("changeDate") final DateTime changeDate,
                         @JsonProperty("changedBy") final String changedBy,
                         @JsonProperty("reasonCode") final String reasonCode,
                         @JsonProperty("comments") final String comments,
-                        @JsonProperty("userToken") final String userToken) {
+                        @JsonProperty("userToken") final String userToken,
+                        @JsonProperty("objectType") final ObjectType objectType,
+                        @JsonProperty("objectId") final UUID objectId) {
```

Comment:
Because we start generating api code based on swagger def, ordering of argument becomes more important to ensure that various api are constructed the same way. I started to [create a page to define some -- arbitrary-- rules](https://github.com/killbill/killbill-swagger-coden/blob/master/SCHEMA_RULES.md#kill-bill-rules):

I would suggest to move args to have:
* ...
* `changeDate`
* **`objectType`**
* **`objectId `**
* `changedBy`
* ...